op-service/metrics/block.go

```diff
@@ -0,0 +1,17 @@
+package metrics
```

Comment:
nit: `block.go` => `proposer.go`, this feels like a more proposer related concept